thrifty-schema/src/main/java/com/microsoft/thrifty/schema/FieldNamingPolicy.java

```diff
@@ -58,4 +77,42 @@ public String apply(String name) {
             return name;
         }
     };
+
+    /**
+     * Find case format from string.
+     * @param s the input String
+     * @return CaseFormat the case format of the string.
+     */
+    @Nullable
+    private static CaseFormat CASE_FORMAT(String s) {
```

Comment:
checkstyle is failing the build over this method name - how about `caseFormatOf` or `parseCaseFormat` or something similar?

Reply:
caseFormatOf looks good - https://github.com/Microsoft/thrifty/pull/165/commits/2472d844ade0a6b2d5a9def89e26dfef1f322e62